client/me/purchases/manage-purchase/change-payment-method/index.jsx

```diff
@@ -387,6 +393,10 @@ function useAssignablePaymentMethods() {
 		activePayButtonText: translate( 'Save card' ),
 	} );
 
+	const payPalMethod = useCreatePayPal( {
+		labelText: translate( 'Add a new PayPal account' ),
```

Comment:
I wonder if "Add a PayPal account" (from https://github.com/Automattic/wp-calypso/issues/47518), or even just "PayPal", would be better wording here for most situations, with "Add a new PayPal account" only used when PayPal is already the current payment method for the subscription (in combination with https://github.com/Automattic/wp-calypso/pull/48583).

It's not a huge difference, but "new" could be a bit confusing if the user already is using PayPal for other subscriptions and plans to use the same PayPal account here, for example.

Reply:
Let's do "PayPal", to keep it similar to Checkout and to match the verb-less credit card label:

<img width="500" alt="Screen Shot 2021-01-05 at 4 29 43 PM" src="https://user-images.githubusercontent.com/942359/103701052-5c1a6880-4f73-11eb-9153-8acf515c186f.png">

Reply:
I'm ok with either, but I think it's worth saying that I had intentionally changed this away from "PayPal" after a previous discussion we had because this button works differently than the other buttons. If you already have PayPal selected, this button will not be checked and will still create a new billing agreement. I don't feel very strongly either way though so I'll change it as suggested here.

Reply:
>I'm ok with either, but I think it's worth saying that I had intentionally changed this away from "PayPal" after a previous discussion we had because this button works differently than the other buttons. If you already have PayPal selected, this button will not be checked and will still create a new billing agreement.

Definitely agree with that part -- that's why I was saying above that it **should** still be "Add a new PayPal account" (or I guess "New PayPal account" if we're doing away with the verbs) in that particular scenario.

It should hopefully be easy to distinguish between the two cases via something like `getCurrentPaymentMethodId( purchase.payment ) === 'paypal'` (as in https://github.com/Automattic/wp-calypso/pull/48583).